src/app/app.component.ts

```diff
@@ -198,51 +200,91 @@ export class AppComponent implements AfterViewInit {
 
   async initializeTreeAsync() {
     this.ref.detectChanges();
-    await this.lfFolderBrowserElement?.nativeElement.initAsync({
-      treeService: this.lfRepoTreeService
-    });
+    let focusedNode;
+    if (this.selectedFolderPath) {
+      let repoId = await this.repoClient.getCurrentRepoId();
+      let focusNodeByPath = await this.repoClient.entriesClient.getEntryByPath({
+          repoId: repoId,
+          fullPath: this.selectedFolderPath
```

Comment:
type focusedNode
let focusedNode: LfRepoTreeNode | undefined;

Also use const where you can

is lint set up on this repo?, there's some missing semi-colons too